src/frontend/magnify/src/components/profile/IdentityForm/IdentityForm.tsx

```diff
@@ -85,11 +93,25 @@ export default function IdentityForm({ id, name, username, email }: IdentityForm
         name: authContext.user?.name ?? '',
         username: authContext.user?.username ?? '',
         email: authContext.user?.email ?? '',
+        language: authContext.user?.language ?? MagnifyLocales.FR,
```

Comment:
If user language property can be blank I don't it is relevant to set a default value to this field.

Reply:
I change to 
```
MagnifyLocales.FR
```